tools/test/examples/examples_lib.py

```diff
@@ -382,27 +382,36 @@ def compile_repos(config, toolchains, targets, profiles, verbose, exp_filter, jo
                     summary_string = "%s %s %s" % (name, target, toolchain)
                     logging.info("Compiling %s" % summary_string)
 
-                    build_command = ["mbed-cli", "compile", "-t", toolchain, "-m", target, "-j", str(jobs)] + (['-vv'] if verbose else [])
-                    if profiles:
-                        for profile in profiles:
-                            build_command.extend(["--profile", profile])
-
-                    logging.info("Executing command '%s'..." % " ".join(build_command))
-                    proc = subprocess.Popen(build_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-
-                    std_out, std_err = proc.communicate()
-                    std_out = std_out.decode()
-                    std_err = std_err.decode()
-                    print ("\n#### STDOUT ####\n%s\n#### STDERR ####\n%s\n#### End of STDOUT/STDERR ####\n" % (std_out,std_err))
-
-                    if proc.returncode:
-                        failures.append(example_summary)
+                    if cmake:
+                        build_command_seq = ["mbed-tools configure -t {} -m {}".format(toolchain,target), "cmake -S . -B CMAKE_BUILD -GNinja", "cmake --build CMAKE_BUILD"]
```

Comment:
How about 
```suggestion
                        build_command_seq = ["mbed-tools build -t {} -m {} -c".format(toolchain, target)]
```
?

Reply:
is `mbed-tools build` works as one single commands now ? to replace `mbed-tools configure` and `cmake ...`
If so I'd be happy to change it.
which version of mbed-tools are required ?

Reply:
The `build` subcommand will do that for you. I am using version `3.4.0`.